crates/lib/src/identity.rs

```diff
@@ -83,9 +83,9 @@ impl Identity {
     /// [0xb0, 0xb1, 0xb2, ...]
     /// ```
     pub const fn from_be_byte_array(bytes: [u8; 32]) -> Self {
-        // SAFETY: The transmute is an implementation of `u256::from_le_bytes`,
+        // SAFETY: The transmute is an implementation of `u256::from_be_bytes`,
```

Comment:
IDT this is true; the transmute is behaving the same here as in `from_byte_array`; the difference is in the `u256::from_le` vs `u256::from_be`.